steward/src/cork/proposals.rs

```diff
@@ -119,6 +119,28 @@ async fn poll_approved_cork_proposals(state: &mut ProposalThreadState) -> Result
             Ok(r) => r.into_inner().proposal,
             Err(err) => {
                 if err.code() == Code::NotFound {
+                    // look ahead in case id was skipped
+                    let mut increments = 0;
```

Comment:
I don't think this can start at zero: we're in the error handler for the query where `proposal_id` is already `state.last_processed_proposal_id + 1`, so if there is a gap you'd have to start by counting the first implicit one.